dev-tools/prod-gen-release-notes-settings.json

```
@@ -0,0 +1,56 @@
+{
+    "$schema": "https://raw.githubusercontent.com/KinsonDigital/kd-admin/preview/schemas/gen-release-notes-schema.json",
+    "ownerName": "KinsonDigital",
+    "repoName": "Infrastructure",
+    "githubTokenEnvVarName": "CICD_TOKEN",
+    "milestoneName": "${VERSION}",
+    "headerText": "${REPONAME} ${RELEASETYPE} Release Notes - ${VERSION}",
+    "releaseType": "Production",
+    "emojisToRemoveFromTitle": [
+        "🚧",
+        "🐛",
+        "✨",
+        "🚀",
+        "🧪",
+        "🔬"
+    ],
+    "issueCategoryLabelMappings": {
+        "New Features ✨": "new-feature",
```

Fix new feature label name:

```suggestion
        "New Features ✨": "✨new-feature",
```